modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -485,15 +495,23 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       (keyShare) => keyShare.from === 'bitgo' && keyShare.to === recipient
     );
     if (!bitGoToRecipientShare) {
-      throw new Error('Missing BitGo to User key share');
+      throw new Error(`Missing BitGo to ${recipient} key share`);
     }
 
-    const backupToUserShare = await encryptNShare(otherShare, recipientIndex, userGpgKey.publicKey, userGpgKey);
+    await this.verifyWalletSignatures(
+      userGpgKey.publicKey,
+      backupGpgKey.publicKey,
+      bitgoKeychain,
+      bitGoToRecipientShare.publicShare,
+      recipientIndex
+    );
```

Comment:
Should we also verify signatures in `createUserKeychainFromThirdPartyBackup` ?

Reply:
yeah, but bitgo (inc) is no long going to hold the backup share.   Trust will, and so should be treated like any other 3rd party.   so this old (nike phase 1) way of doing things should be going away.